pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -840,6 +840,20 @@ func removeTerminatedContainers(containers []*runtimeapi.Container) []*runtimeap
 	return result
 }
 
+func cpuStatsInitialized(cpu *statsapi.CPUStats) bool {
+	return cpu != nil &&
+		(*cpu.UsageCoreNanoSeconds != 0 || *cpu.UsageNanoCores != 0)
+}
+
+func memoryStatsInitialized(memory *statsapi.MemoryStats) bool {
+	return memory != nil &&
+		(*memory.UsageBytes != 0 ||
```

Comment:
Similar to CPU stats, why we use the actual number instead of the Time metav1.Time of MemoryStats ?